functionary/ui/views/view_base.py

```diff
@@ -60,4 +60,4 @@ def test_func(self):
 class PermissionedFormUpdateView(LoginRequiredMixin, UserPassesTestMixin, UpdateView):
     def test_func(self):
         env = self.request.session.get("environment_id")
-        return self.request.user.has_perm(Permission.ENVIRONMENT_READ, env)
+        return self.request.user.has_perm(Permission.ENVIRONMENT_CREATE, env)
```

Comment:
This should be changed back for now as well.

Reply:
Fixed